modules/nw-networkoperator-crd.adoc

```diff
@@ -0,0 +1,99 @@
+// Module filename: nw-networkoperator-crd.adoc
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
+
+[id="network-networkoperator-default-crd-{context}"]
+= Network Operator custom resource
+
+The `NetworkConfig` configuration in a `networkoperator.openshift.io` Custom
+Resource (CR) object stores the advanced settings for the `Network Operator`.
+
+The following parameters are set in the `config.openshift.io` CR and the
+installation program copies them to the `networkoperator.openshift.io` CR object
+before the cluster is created.
+
+* `cidr`
+* `hostPrefix`
+* `networkType`
+* `serviceNetwork`
+
+The following CR displays the default configuration for the Network Operator and
+explains both the parameters you can configure and valid parameter values:
+
+.Default NetworkConfig CR
+[source,yaml]
+----
+apiVersion: networkoperator.openshift.io/v1
+kind: NetworkConfig
+spec:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14 <1>
+    hostSubnetLength: 9 <2>
+  serviceNetwork: 172.30.0.0/16 <3>
+  defaultNetwork: <4>
+    type: OpenShiftSDN <5>
+    openshiftSDNConfig:
```

Comment:
you probably need a footnote on the `openshiftSDNConfig` line, clarifying that this section needs to be specified if (and only if) you want to override any of the OpenShift SDN configuration